src/jsx.d.ts

```diff
@@ -2021,7 +2021,69 @@ export namespace JSXInternal {
 			| SignalLike<number | string | undefined>;
 	}
 
-	export interface IntrinsicElements {
+	export interface SVGElements {
```

Comment:
```suggestion
	export interface SVGInstrinsicElements {
```

or maybe `IntrinsicSVGElements`